core/templates/pages/story-editor-page/services/story-editor-state.service.ts

```diff
@@ -108,6 +109,18 @@ angular.module('oppia').factory('StoryEditorStateService', [
         return _storyIsInitialized;
       },
 
+      setExpIdChanged: function() {
```

Comment:
I'm not sure about the context of this function i.e, whether it will respond the expID is changed in the node of a story OR one or more expIDs are changed in the story? If it's the latter one, then I would suggest renaming it to  `setExpIdsChanged`.

Reply:
It is the latter, renamed.